src/components/metadataFormControls/ModelSingleSelect/ModelSingleSelectField.tsx

```diff
@@ -71,3 +77,32 @@ export function ModelSingleSelectField<TModel extends DisplayableModel>({
         </Field>
     )
 }
+
+export function ModelSingleSelectFormField<TModel extends DisplayableModel>({
```

Comment:
I wanted to remove the "useField" within `ModelSingleSelectField`. So that `ModelSingleSelect` follows the API like`FF`-Fields from ui (eg InputFieldFF`). This is now composes that with `useField` so it could be used the same way with just a name-refactor.